python/lsst/rubintv/production/allSky.py

```diff
@@ -436,6 +443,12 @@ def uploadLastStill(self, convertedFiles):
             self.uploader.googleUpload(channel=channel,
                                        sourceFilename=sourceFilename,
                                        uploadAsFilename=uploadAsFilename)
+            try:
```

Comment:
This part looks like an independent change from adding epoUploader too?

Reply:
I think this comment is outdated now, but GitHub somehow isn't showing that to be the case. I think you're referring to the fact that this used to be using the main uploader to send the still, which was wrong. It's now using the EPO uploader to send the still to them as well, and so is part of the change, I just don't know why GitHub hasn't caught that that's been updated already (maybe because the comment is on the `try` line and that line itself didn't change, I guess). But yes, it's all part of the same thing.